app/models/event.rb

```diff
@@ -116,6 +119,19 @@ def completed?
     status == "Completed"
   end
 
+  def use_template
```

Comment:
What do you think about naming this `use_as_template`? Is that what we're doing here?